runtime/queries/bash/highlights.scm

```diff
@@ -60,7 +48,6 @@
   ">>"
   "<"
   "|"
-  (expansion_flags)
```

Comment:
Apart from this change, it should be kept since `expansion_flags` was removed upstream